packages/devextreme-scss/scss/widgets/base/dataGrid/_index.scss

```diff
@@ -205,6 +205,26 @@ $datagrid-text-stub-background-image-path: null !default;
     }
   }
 
+  .dx-datagrid-total-footer {
+    .dx-row.dx-footer-row {
+      td.dx-command-expand {
+        &.dx-datagrid-sticky-column, &.dx-datagrid-sticky-column-left, &.dx-datagrid-sticky-column-right {
+          background-color: $datagrid-base-background-color;
+        }
+      }
+
+      .dx-datagrid-sticky-column,
+      .dx-datagrid-sticky-column-left,
+      .dx-datagrid-sticky-column-right {
+        padding-inline: 0;
+
+        .dx-datagrid-summary-item {
+          background-color: $datagrid-base-background-color;
+        }
```

Comment:
is it possible to move `padding-inline: $datagrid-cell-padding;` line from [here](https://github.com/DevExpress/DevExtreme/pull/30481/files) to this block?

I think in that way we could remove that huge selector and keep related styles in one place